plugins/module_utils/netbox_utils.py

```diff
@@ -392,6 +395,7 @@
     "vlan_groups",
 }
 
+
 NETBOX_ARG_SPEC = dict(
```

Comment:
Can you add to the **REQUIRED_ID_FIND** for `protocol` and status and then create the constant to convert the status to an integer?

This eventually won't be needed, but at least for now.

Reply:
Dict already exist, so I only changed the value type.